source/dub/dub.d

```diff
@@ -655,7 +678,7 @@ class Dub {
 		}
 
 		if (existing) {
-			if (!ver.startsWith("~") || !(options & FetchOptions.forceBranchUpgrade) || location == PlacementLocation.local) {
+			if (!ver.startsWith("~") || !(options & FetchOptions.forceBranchUpgrade)/* || location == PlacementLocation.local*/) {
```

Comment:
I commented this out because I didn't want to have to deal with it for now, will fix it, but first perhaps someone could explain why this is special-cased for local?